src/Map/src/Twig/UXMapComponent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Twig;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
+ */
+final class UXMapComponent
+{
+    public int $zoom;
+
+    public array $center;
+
+    public array $markers;
+}
```

Comment:
Can be typed with PHPDoc

Reply:
Add types (this is purely informational for now, as the component is bypassed at runtime by the listener)